instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import PostContainer from '/Users/lauradaugherty/Lambda Projects/React-Insta-Clone/instagram/src/components/PostContainer/PostContainer.js'
```

Comment:
These 3 imports are incorrectly written. The directions are navigating through your computer. Not through the project itself! Should be like this: 

import PostContainer from '../PostContainer/PostContainer'
import SearchBar from '../SearchBar/SearchBar'
import dummyData from '../../dummy-data';